10x/count/sample_def.json

```diff
@@ -0,0 +1 @@
+[{"gem_group": null, "lanes": null, "sample_names": ["hgmm_100"], "sample_indices": ["any"], "fastq_mode": "ILMN_BCL2FASTQ", "read_path": "read_path"}]
```

Comment:
Could you change this to a pretty printed version? It's hard to read this way.

Reply:
Yep, fixed.